docs/getting-started/what-is-dotnet.md

```diff
@@ -1,6 +1,155 @@
+
+- [F#](https://channel9.msdn.com/Events/Build/2016/T661 "F#").
+ 
+(At the moment, other languages are not supported, though that will change going forward.) 
+
+You have a wide choice of development environments for writing your app, ranging from a simple text editor or programmer's editor of your choice to [Visual Studio Code](https://code.visualstudio.com/ "Visual Studio Code"), to [Visual Studio 2015](https://www.visualstudio.com/en-us/visual-studio-homepage-vs.aspx "Visual Studio 2015").   
+
+# What's Different about .NET Core?  #
+
+Let's take a high-level view of some of the components and features that .NET Core makes available. 
+
+## Cross-Platform Support ##
+
+.NET Core apps can run on both the 32-bit and 64-bit Windows platforms, as well as on Mac OSX and Linux.
+
+In contrast, apps that target the full .NET Framework can run only on Windows. And UWP apps, which use an implementation of .NET Core, can run only on Windows desktop and tablets, as well as on Windows Phone.
+
+## Open Source ##
+ 
+.NET Core -- both its runtime and its class libraries -- are open source. In addition to making the source code for .NET Core freely available, this means that:
+
+- Design notes, feature specifications, and implementation-specific documentation is publicly available. 
+
+- Code reviews of .NET Core features are public.
+
+- By using GitHub's pull request model, you can open bugs, suggest modifications to source code, and submit new feature requests and source code.
+
+- You can download all of the source code and build it on your own system. The .NET Framework libraries, the runtime, and .NET Framework tools can be built on any of the supported platforms (that is, on Windows x86/x64, Max OS/X, and Linux). 
+
+## Console Applications ##
+
+The .NET Framework on the Windows desktop allows you to create console applications, and so does .NET Core. But .NET Core reinvigorates the console application:
+
+- Console applications can be compiled to run cross-platform on Windows, the Macintosh, and Linux.
+
+- Console applications can be compiled natively. This combines the benefits of a managed application with the performance of a native C/C++ application. 
+
+## Ease of Installation ##
+
+Because .NET Core is not an operating system component, installation does not require administrator privileges, and it does not touch system components like operating system directories or, on Windows systems, the Windows Registry. Installation is as simple as copying some files to a directory on the target computer or compiling the framework natively into the application. 
+
+New developers can get started developing for .NET in under one minute, including downloading the frameworks and tools. You can install .NET Core from the [.NET Core getting started page](https://dotnet.github.io/getting-started/#/windows ".NET Core getting started page"). 
```

Comment:
I think the link should be just "https://dotnet.github.io/getting-started".  The '#/windows" gets added by client side routing once the browser detects your OS.